packages/main/src/plugin/cancellation-token.ts

```diff
@@ -33,8 +32,7 @@ const shortcutEvent: extensionApi.Event<any> = Object.freeze(function (callback,
 
 export class CancellationTokenImpl implements extensionApi.CancellationToken {
   private _isCancellationRequested: boolean;
-  // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  private emitter: Emitter<any> | undefined;
+  private emitter: Emitter<unknown> | undefined;
```

Comment:
We only emit `undefined`

https://github.com/podman-desktop/podman-desktop/blob/0698e4d2fd1f3fcb64bfc8d66a96003545b6ee89/packages/main/src/plugin/cancellation-token.ts#L47

Should we use `<void>` instead ? so we can simplify 

If we use void, we may change the extension-api too ?

https://github.com/podman-desktop/podman-desktop/blob/be499999e49627d74cad14efe98c9dbe3fed1111/packages/extension-api/src/extension-api.d.ts#L1315

Reply:
we won't be able to change anything in the extension-api.d.ts